xrpVendordep/src/main/java/edu/wpi/first/wpilibj/xrp/XRPRangefinder.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj.xrp;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+
+/** This class represents the ultrasonic rangefinder on an XRP robot. */
+public class XRPRangefinder {
+  private final AnalogInput m_rangefinder = new AnalogInput(2);
+
+  /**
+   * Get the measured distance in metres. Distance further than 4m will be reported as 4m.
+   *
+   * @return distance in metres
+   */
+  public double getDistanceMetres() {
+    return (m_rangefinder.getVoltage() / 5.0) * 4.0;
+  }
+
+  /**
+   * Get the measured distance in inches.
+   *
+   * @return distance in inches
+   */
+  public double getDistanceInches() {
+    return getDistanceMetres() * 39.3701;
```

Comment:
```suggestion
    return Units.metersToInches(getDistanceMeters());
```
Import `edu.wpi.first.math.util.Units` for this function.